index.d.ts

```diff
@@ -47,8 +47,11 @@ export type ResponseHandler = {
   (request: FakeXMLHttpRequest & ExtraRequestData):
     | ResponseData
     | PromiseLike<ResponseData>;
+};
+
+export type ResponseHandlerInstance = ResponseHandler & { 
```

Comment:
Can we rename `ResponseHandler` to `ResponseHandlerFunc`, and just use `ResponseHandler` for the instances that have `async` and `numberOfCalls`?

Reply:
Hmm I take it back, looks like PR #317 actually broke ResponseHandler